<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de credores</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Credores cadastrados</string>
    </property>
    <widget class="QLineEdit" name="txtPesquisarCredor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtPesquisarCredor{
border:none;
border-bottom:1px solid #f27e13;
padding:5px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pesquisar..</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPesquisarCredor">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnPesquisarCredor{
border:1px solid #f27e13;
padding:5px;
background-color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="dgvCredores">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>391</width>
       <height>141</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Credor</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="btnAlterar"/>
    <addaction name="btnExcluir"/>
    <addaction name="btnSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="btnAlterar">
   <property name="text">
    <string>Alterar</string>
   </property>
  </action>
  <action name="btnExcluir">
   <property name="text">
    <string>Excluir</string>
   </property>
  </action>
  <action name="btnSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
